play.pokemonshowdown.com/style/client2.css

```diff
@@ -1110,6 +1110,11 @@ p.or:after {
 .revealed {
 	background-color: #C8FDEA;
 }
+.chat small {
+	font-weight: normal;
+	font-size: 8pt;
+	color: #888888;
```

Comment:
I actually intentionally removed this CSS so not all small things would automatically be grey. We should do an actual `.grey` class, to make stuff grey.

Reply:
`.gray` won the poll though. By the way `.gray` should probably go in `battle-log.css`, not here.

Reply:
(Poll closes in half a day, I'm tentatively waiting until then to merge.)